unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Video/Plugins/WebVideoPlayer.cs

```diff
@@ -61,8 +67,8 @@ public void Update()
 
                     break;
                 case VideoState.PLAYING:
-                    if (visible)
-                        plugin.TextureUpdate(videoPlayerId);
+                    //if (visible)
+                    //    plugin.TextureUpdate(videoPlayerId);
```

Comment:
Same here

Reply:
As stated before, this is a test for QA, last changes will be reverted after approval